core/templates/pages/exploration-player-page/services/answer-classification.service.spec.ts

```diff
@@ -73,6 +79,7 @@ describe('Answer Classification Service', () => {
       ).and.returnValue(false);
       spyOn(appService, 'isMachineLearningClassificationEnabled')
         .and.returnValue(false);
+      stateClassifierMappingService.init('0', 0);
```

Comment:
Here and elsewhere in this file, could you please change '0' to 'exp_id'? Hard to tell that it's an exploration ID just from reading it.

Reply:
Done